src/Sentry/SentryOptions.cs

```diff
@@ -383,6 +383,14 @@ public bool Debug
         /// </remarks>
         public TimeSpan CacheFlushTimeout { get; set; } = TimeSpan.FromSeconds(1);
 
+        /// <summary>
+        /// Defaults tags to add to all events. (These are indexed by Sentry).
+        /// </summary>
+        /// <remarks>
+        /// If the key already exists in the event, it will not be overwritten by a default tag.
+        /// </remarks>
+        public Dictionary<string, string> DefaultTags { get; } = new Dictionary<string, string>();
```

Comment:
To avoid allocating this map as a result of instantiating this class, could have a private backing field, nullable, and on the getter here we set instance?

This way we only create on the first use.

As this SDK becomes more general purpose I want to be mindful of the initial cost you pay just to `Init` Sentry on a basic use case.


Something like
```suggestion
        public Dictionary<string, string> DefaultTags => _defaultTags =?? new Dictionary();
```

Reply:
Ah yes! good use of the null-coalescing operator.